docs/javascript/guidelines.rst

```diff
@@ -5,6 +5,7 @@ Guidelines
 * Use camelCase for variables and **not** underscores or dashes
 * Use **dot** annotation ``.`` for javascript file naming
 * Use **single-quotes** ``'`` for **all** values
+* Use a maximum length of 120 characters for JS code
```

Comment:
We decided on first getting to 120 lines and than limit it to 80 lines (for js and css) except for "allowRegex" and "allowUrlComments"